__tests__/index.test.js

```
@@ -775,4 +776,28 @@ describe("Purchases", () => {
         });
       });
   });
+
+  describe("isConfigured", () => {
```

shouldn't we also test that we throw when it's not configured?